README.md

```diff
@@ -355,6 +401,35 @@ functionality to PHP before 5.5, so if you'd like process titles updated,
 install the PECL module as well. php-resque will automatically detect and use
 it.
 
+### Resque Scheduler
+
+resque-scheduler requires a special worker that runs in the background. This
+worker is responsible for pulling items off the schedule/delayed queue and adding
+them to the queue for resque. This means that for delayed or scheduled jobs to be
+executed, that worker needs to be running.
+
+A basic "up-and-running" `resque-scheduler.php` file that sets up a
+running worker environment is included (`vendor/resque-scheduler.php` when
+installed via composer). It accepts many of the same environment variables as
+the main workers for php-resque:
+
+* `REDIS_BACKEND` - Redis server to connect to
+* `LOGGING` - Enable logging to STDOUT
+* `VERBOSE` - Enable verbose logging
+* `VVERBOSE` - Enable very verbose logging
+* `INTERVAL` - Sleep for this long before checking scheduled/delayed queues
+* `APP_INCLUDE` - Include this file when starting (to launch your app)
+* `PIDFILE` - Write the PID of the worker out to this file
+
+The resque-scheduler worker requires resque to function. The demo
+resque-scheduler.php worker allows you to supply a `RESQUE_PHP` environment
+variable with the path to Resque.php. If not supplied and resque is not already
+loaded, resque-scheduler will attempt to load it from your include path
+(`require_once 'Resque/Resque.php';'`)
```

Comment:
We may not need this anymore? Since Scheduler is now embedded within Resque itself, they won't be separated...